src/compiler/crystal/tools/expand.cr

```diff
@@ -140,23 +140,21 @@ module Crystal
           else
             @message = "no expansion found: #{node} may not be a macro"
           end
-          false
-        else
-          contains_target(node)
+          return false
         end
       end
+      contains_target(node)
```

Comment:
**question:** This looks like it's actually a semantic change.
Before this change, the method ould return `nil` when `node.location` is nil.  I suppose this is wrong and changing it to `true` (which `contains_target` does) is probably correct.
However, I think this bug fix should be separate from the return type refactoring.

Ditto below.